### PR TITLE
cmd ref: doc metrics diff --precision

### PR DESCRIPTION
**Pull request by efiop**
Per https://github.com/iterative/dvc/pull/4026

> You may disregard these recommendations if you used the **Edit on GitHub** button from dvc.org to improve a doc in place.

❗ Please read the guidelines in the [Contributing to the Documentation](https://dvc.org/doc/user-guide/contributing/docs) list if you make any substantial changes to the documentation or JS engine.

🐛 Please make sure to mention `Fix #issue` (if applicable) in the description of the PR. This causes GitHub to close it automatically when the PR is merged.

Please choose to [allow us](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to edit your branch when creating the PR.

Thank you for the contribution - we'll try to review it as soon as possible. 🙏

### CODE REVIEW

File: content/docs/command-reference/metrics/diff.md [hunk 1]
```diff
@@ -58,6 +59,9 @@ lists all the current metrics without comparisons.
   useful when only one metrics file is in use or there is no intersection
   between the metric names.
 
+- `--precision <n>` - round metrics to n decimal digits. Rounds to 5 digits by
```
Comment (shcheklein) [inline content/docs/command-reference/metrics/diff.md:62]: 💅 n -> `n`

Comment (shcheklein) [reply content/docs/command-reference/metrics/diff.md:62]: ideally, `Rounds to 5 digits default` should be very explicitly mentioned in the description

Comment (efiop) [reply content/docs/command-reference/metrics/diff.md:62]: I guess `n` -> `<n>` actually. At least that's how `git` does it.


> ideally, Rounds to 5 digits default should be very explicitly mentioned in the description

Could you reword, please? I didn't get your point.

Comment (shcheklein) [reply content/docs/command-reference/metrics/diff.md:62]: `I guess n -> <n> actually. ` - fine with either, just some highlighting is needed here it feels like

> Could you reword, please? I didn't get your point.

yeah, I think my point is if we always cut info at 5 decimal digits, it's better to mention in the description section above (not only here, in the `--precision` description).

Btw, do we apply `--precision` to all the columns? or only to the diff one? would be useful to be clear on this as well.

Comment (efiop) [reply content/docs/command-reference/metrics/diff.md]: @shcheklein Thanks! Adjusted.

Comment (shcheklein) [reply content/docs/command-reference/metrics/diff.md]: Good! thanks 🙏. I actually meant n in the `n decimal digits` part :)

Comment (efiop) [reply content/docs/command-reference/metrics/diff.md]: @shcheklein Sorry, I might be having a smooth-brain moment right now, but I don't get what you mean.

Comment (shcheklein) [reply content/docs/command-reference/metrics/diff.md]: :)

round metrics to n decimal digits

--->

round metrics to `n` decimal digits

Comment (efiop) [reply content/docs/command-reference/metrics/diff.md]: I've used `<n>`. But ok, using backticks now. Thanks!

**Comment by jorgeorpinel** [comment]
Thanks for this!

**Pull request merged**